### PR TITLE
[WIP] Core/DataStores: Update DB2 Structures

### PR DESCRIPTION
**Pull request by funjoker**
**Changes proposed:**

-  Name Unknowns

**Target branch(es):** 3.3.5/master

- [x] master

**Issues addressed:** Closes #  (insert issue tracker number)

**Tests performed:** (Does it build, tested in-game, etc.)

**Known issues and TODO list:** (add/remove lines as needed)

- [ ] Add unlisted Structures

### CODE REVIEW

**Comment by Shauren** [comment]
source? also I don't want these unused structures - all it does is get outdated quickly and mess with DB2LoadInfo generator

**Comment by Selenium** [comment]
@Shauren Latest BFA build https://github.com/Arctium/ClientDB-Structures/commit/f4c470421e5f3b36b629e1d5c26778f5f7935177

File: src/server/scripts/Outland/BlackTemple/boss_illidan.cpp [hunk 1]
```diff
@@ -143,7 +143,7 @@ enum Misc
 };
 
 /**** Creature Summon and Recognition IDs ****/
-enum CreatureEntry
+enum IllidanCreatureEntry
```
Comment (Kittnz) [inline src/server/scripts/Outland/BlackTemple/boss_illidan.cpp]: sneaky sneaky

Comment (funjoker) [reply src/server/scripts/Outland/BlackTemple/boss_illidan.cpp]: :D

**Comment by funjoker** [comment]
Not from Fabi,
from
https://github.com/Marlamin/WoWDBDefs/tree/master/definitions

@Malamin <3

**Comment by funjoker** [comment]
Well if we can now name all structs for 7.3.5 (prolly last patch before BfA).
The theoretical "7.3.5" branch is not changed again there.

And for now some DB2 might be useful and people might start workin on some systems which needed DB2 data.

**Comment by Traesh** [comment]
The problem is that when TC will jump to BfA you'll force the guy doing it (aka Shauren) to update a lot of unused Struct to keep it up to date

**Comment by Shauren** [comment]
Unused structures are not going in - they mess with https://github.com/Shauren/WowClientDB2MySQLTableGenerator

**Comment by funjoker** [comment]
So people who need these structs can find them 
https://github.com/Marlamin/WoWDBDefs/tree/master/definitions
or
Arctium/ClientDB-Structures@f4c4704

**Pull request closed without merging**